src/util/check.h

```diff
@@ -5,30 +5,23 @@
 #ifndef BITCOIN_UTIL_CHECK_H
 #define BITCOIN_UTIL_CHECK_H
 
-#if defined(HAVE_CONFIG_H)
```

Comment:
Any idea what this bitcoin-config.h was needed for before?

Reply:
This was added without explanation in commit 691c817b340 by @ryanofsky

Reply:
> This was added without explanation in commit [691c817](https://github.com/bitcoin/bitcoin/commit/691c817b340d10e806dc3b1834d2a8fcc5e681fd) by @ryanofsky

I added the include there to define the PACKAGE_BUGREPORT macro referenced below (it was probably needed previously, but not obvious because of an earlier include)